R/control.R

```diff
@@ -79,9 +79,6 @@ spim_control <- function(short_run, n_chains, deterministic = FALSE,
     ## Disable early exit, if it's been set up, as we also don't support that
     pmcmc$filter_early_exit <- FALSE
 
-    ## Disable restart, as the deterministic fit does not support this yet
-    pmcmc$save_restart <- NULL
-
```

Comment:
below you can make the n_workers line

```
pmcmc$n_workers <- min(pmcmc$n_chains, pmcmc_n_threads_total)
```

I think, and it should do the right thing - you might confirm on your laptop though please

Reply:
Changed and tested successfully!

Reply:
The tests are failing below, likely because things are not current with master I believe, you might want to test after bringing things back up to date